docs/what-if/add-test-content.md

```diff
@@ -0,0 +1,18 @@
+# What If I want to add test content
+
+- Change `application` to `development` in the [autoexport-module.yaml](../../repository-data/application/src/main/resources/hcm-config/configuration/modules/autoexport-module.yaml) file:
+
+```YAML
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:modules/autoexport:
+      /hippo:moduleconfig:
+        autoexport:modules:
+        - repository-data/development:/
+```
+
+- After starting up the application login to the CMS, and **switch Autoexport on**
```

Comment:
Could use adding a remark to make sure to not to commit `repository-data/development/src/main/resources/hcm-config/main.yaml` with the auto-export on as switching Autoexport in CMS automatically propagates this status to that file.

Reply:
The remark is still missing from the readme file, @robValtech - the phrase `Only track and commit the file changes that happened under the 'development' folder` is not enough on its own to protect from accidental turning the auto export on because the `main.yaml` file is itself under the `development` folder.

Reply:
@ziemsky We're never supposed to commit any changes to the `main.yaml` file :)

Reply:
Ah, I see. As long as it's clear to everyone involved I have no objections :wink: